serializer.rst

```diff
@@ -223,7 +222,7 @@ A service leveraging `APCu`_ (and APC for PHP < 5.5) is built-in.
         framework:
             # ...
             serializer:
-                cache: serializer.mapping.cache.apc
+                cache: id.of.my.own.cache.service
```

Comment:
Instead of using the `framework.serializer.cache` option we should explain to configure the `cache.serializer` (which defaults to `cache.system`) service through the `framework.cache.pools` option.